docs/share.md

```diff
@@ -12,19 +12,28 @@ You can share the data your connector accesses by connecting to the data, creati
 
 Or, if you prefer to have your users test the connector, you can place the connector files in the directories and then run the commands as follows:
 
-**Tableau Desktop:** This is the same as developing the connector.
-Place the files into a subdirectory you create, and then use the -DConnectPluginsPath command line argument.  For example, place your plugin files in `C:\tableau_connectors\myconnector` and then start Tableau:
+**Tableau Desktop:** 
+
+1. Place the files into a subdirectory you create.
+1. Use the `-DConnectPluginsPath` command line argument.  
+
+For example, place your connector files in `C:\tableau_connectors\myconnector` and then start Tableau:
 
 ```
 tableau.exe -DConnectPluginsPath=C:\tableau_connectors
 ```
 
-**Tableau Server:** Follow the same instructions as Tableau Desktop for each server node.
-Then set this option: "native_api.connect_plugins_path: C:/tableau_connectors".
-For information about using TSM to do this, see [tsm configuration set Options](https://onlinehelp.tableau.com/current/server-linux/en-us/cli_configuration-set_tsm.htm) in the Tableau Server help.
+**Tableau Server:** 
+
+1. Follow the same instructions as Tableau Desktop for each server node.
+1. Set this option: `native_api.connect_plugins_path: C:/tableau_connectors`
+
+For information about using TSM to set the option, see [tsm configuration set Options](https://onlinehelp.tableau.com/current/server-linux/en-us/cli_configuration-set_tsm.htm) in the Tableau Server help.
 
 For example:
 
 ```
-tsm configuration set -k native_api.connect_plugins_path -v C:/tableau_connectors
+tsm configuration set -k native_api.connect_plugins_path -v C:/tableau_connectors 
 ```
+
+If you get a configuration error, try adding the `--force-keys` option to the end of the command.
```

Comment:
Not sure where, but we may want to mention that for the server to see changes to the connectors, it must be restarted. For example, if you drag and drop a new connector into the server's connector directory, it will not automatically show up until the server is restarted. I can see that maybe confusing some people wondering why their brand new connector isn't showing up.

Reply:
I added Restart server as the last step in the Tableau Server section.